servers/assessment/assessments/router.go

```diff
@@ -0,0 +1,103 @@
+package assessments
+
+import (
+	"net/http"
+
+	"github.com/gin-gonic/gin"
+	"github.com/google/uuid"
+	promptSDK "github.com/ls1intum/prompt-sdk"
+	"github.com/ls1intum/prompt2/servers/assessment/assessments/assessmentDTO"
+	log "github.com/sirupsen/logrus"
+)
+
+func setupAssessmentRouter(routerGroup *gin.RouterGroup, authMiddleware func(allowedRoles ...string) gin.HandlerFunc) {
+	assessmentRouter := routerGroup.Group("/assessment")
+
+	assessmentRouter.GET("/:courseParticipationID/:coursePhaseID", authMiddleware(promptSDK.PromptAdmin, promptSDK.CourseLecturer), getAssessmentsForStudentInPhase)
```

Comment:
Remove the :coursePhaseID here. You already have that in main. This will lead to issues
```suggestion
	assessmentRouter.GET("/:courseParticipationID", authMiddleware(promptSDK.PromptAdmin, promptSDK.CourseLecturer), getAssessmentsForStudentInPhase)
```

Reply:
Good Catch